wire-ios/Wire-iOS/Sources/Managers/DatadogWrapper.swift

```diff
@@ -164,6 +167,14 @@ extension RemoteMonitoring.Level {
 
 extension DatadogWrapper: WireSystem.LoggerProtocol {
 
+    public func prepareMessage<Payload>(title: String, payload: Payload) -> String where Payload : Encodable {
+        guard let payloadData = try? payloadEncoder.encode(payload),
+              let payloadString = String(data: payloadData, encoding: .utf8) else {
+          return "\(title): PAYLOAD ENCODING ERROR"
+        }
+        return "\(title): \(payloadString)"
+    }
```

Comment:
This is a duplication of the method in the `WireLogger`, and this method is never actually called. I would argue that we don't need this protocol requirement.

Reply:
I would suggest to move it to some other protocol, or do the encoding in the message itself, or remove it from protocol and leave only on WireLogger if we won't use it on DatadogWrapper.
This method is called only on WireLogger currently.